omnigibson/systems/system_base.py

```diff
@@ -79,14 +36,20 @@ def __init_subclass__(cls, **kwargs):
         if cls._register_system:
             REGISTERED_SYSTEMS[cls.__name__] = cls
 
-    # Simulator reference
-    simulator = None
+        cls._uuid = abs(hash(cls.__name__)) % (10 ** 8)
```

Comment:
can we make this a standalone utility function in python utils or something? so we don't have to duplicate our hashing code

Reply:
done